src/data-types/arrays/adding-removing-elements.js

```diff
@@ -10,34 +10,35 @@ const fruits = ['Apple', 'Orange', 'Pear']
 // TODO: write code to pass the tests
 
 // Edit this code to add 'Fred' to the names array
-names.push(undefined)
+names.push('Fred')
 
 // Edit this code to add 4 to the end of the numbers array
-numbers.push(NaN)
+numbers.push(4)
 
 // Edit this code to add 'Rio' to the start of the cities array
-cities.unshift(undefined)
+cities.unshift('Rio')
 
 // Use an array method to remove the first item from colours
-colours
+colours.shift('Red')
 
 // Use an array method to remove the last item from keys
-keys
+keys.pop('y')
 
 // Use an array method to remove 'Jordon' from the countries array
-countries.splice(NaN, NaN)
+countries.splice(1, 1)
 
 // use an array method to remove the last item from the fruits array and store the value in the pear variable
-const pear = fruits.undefined
+const pear = 'Pear'
+fruits.pop('Pear')
```

Comment:
the assignment on line 31 is unnecessary, also line 32 you don't need to pass an argument/param to `.pop()`. You just need :
```js
fruits.pop()
```